worker-markup-container-fs/src/main/config/cfg~caf~worker~MarkupWorkerConfiguration.js

```diff
@@ -30,5 +30,16 @@
         "On": ["Le", "W dniu", "Op", "Am", "På", "Den"],
         "Separator": ["użytkownik"],
         "Wrote": ["a écrit", "napisał", "schreef", "verzond", "geschreven", "schrieb", "skrev"]
-    }
```

Comment:
As discussed, I don't think we should add these to the configuration by default.

Reply:
Done.